src/Pact/Native.hs

```diff
@@ -64,22 +66,30 @@ import Control.Monad
 import Control.Monad.IO.Class
 import qualified Data.Attoparsec.Text as AP
 import Data.Bifunctor (first)
+import Data.Binary (get, put)
+import Data.Binary.Get (Get, runGetOrFail, getByteString, isEmpty)
+import Data.Binary.Put (Put, runPut, putByteString)
```

Comment:
@edmundnoble should they create a wrapper for `Get` and `Put` similar to our `Chainweb.Utils.Serialization`?

Reply:
Frankly I might prefer that but it seems like a big change.

Reply:
Agreed, that would be a nice refactor but I don't want to hold up the PR for it.